src/components/CouponDetails/index.jsx

```diff
@@ -32,7 +32,7 @@ class CouponDetails extends React.Component {
         },
         {
           label: 'Code',
-          key: 'title',
+          key: 'code',
```

Comment:
This is a fix. API service actually returns `code` column for code details endpoint.

Reply:
**edit**: This will be covered in https://github.com/edx/edx-portal/pull/153 along with further fixes needed.